Formula/arangodb.rb

```diff
@@ -12,7 +13,7 @@ class Arangodb < Formula
 
   depends_on "ccache" => :build
   depends_on "cmake" => :build
-  depends_on "go" => :build
+  depends_on "go@1.13" => :build
```

Comment:
@KVS85 Does this fail to build with `go`?

Reply:
For some reason - yes. Jenkins output stated "Go 1.13 required" before (in Catalina and Mojave which we only support from our side). Maybe it's just a matter of image and what "go" is there (should be 1.14 as far as I can see).

Reply:
We didn't test it with 1.14.

Reply:
If it fails without on our Jenkins: no problem 👍. Just note in future we may not always have the version of your choice available.